internal/pkg/deploy/cloudformation/app.go

```diff
@@ -214,29 +229,29 @@ func (cf CloudFormation) RemoveServiceFromApp(app *config.Application, svcName s
 	}
 
 	// We'll generate a new list of Accounts to remove the account associated
-	// with the input service to be removed.
-	var svcList []string
-	shouldRemoveSvc := false
-	for _, svc := range previouslyDeployedConfig.Services {
-		if svc == svcName {
-			shouldRemoveSvc = true
+	// with the input workload to be removed.
+	var wlList []string
+	shouldRemoveWl := false
+	for _, wl := range previouslyDeployedConfig.Services {
```

Comment:
Should we make this to be `Workloads`?

Reply:
I think yes, but this means we'll have a different name for the Services Cloudformation Metadata key in the App CF template and the struct member that it unmarshals into. I think a lot of app config was written with the assumption that we would only deploy services, and some CF keys aren't service-agnostic. 

https://github.com/aws/copilot-cli/blob/mainline/templates/app/cf.yml#L9 for what i'm talking about

Reply:
I also think this config refactor can be a separate PR.

Reply:
oh got it. Seems like there are a lot to change. Maybe in future PRs then!

Reply:
New Issue here: https://github.com/aws/copilot-cli/issues/1443